Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -945,9 +946,12 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
         super.onActivityResult(requestCode, resultCode, data);
         switch (requestCode) {
             case IMAGE_CAPTURE_REQ_CODE:
+                CommonJsInterface.isUploadPopupOpen = false;
```

Comment:
CommonJsInterface.isUploadPopupOpen = false;

Why this line is added three times?